Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec puma -C ./config/puma.rb
+web: bundle exec rails server
```

Comment:
It wasn't creating the `tmp/pids/server.pid` properly when I ran using `bundle exec puma`